Makefile

```diff
@@ -21,6 +21,12 @@ endif
 
 build: handler.zip
 
+package: build
+	sam package \
+		--s3-bucket buildkite-sar-us-east-1 \
```

Comment:
This is just a bucket in my test account, we’d set up proper CI/CD for the template and SAR resource before merging this